services/cargohold/src/CargoHold/S3.hs

```diff
@@ -221,12 +242,46 @@ signedURL path = do
     toUri x = case parseURI strictURIParserOptions x of
       Left e -> do
         Log.err $
-          "remote" .= val "S3"
+          "remote"
+            .= val "S3"
             ~~ msg (val "Failed to generate a signed URI")
             ~~ msg (show e)
```

Comment:
interesting.  i didn't know `msg` was self-concatenating.

Reply:
It actually is: https://hoogle.zinfra.io/file/nix/store/m3hbxxbjk8j6a13gym6vhswdqx2z8a89-tinylog-0.15.0-doc/share/doc/tinylog-0.15.0/html/src/System.Logger.Message.html#msg. But, I have to admit I didn't knew it either :smile_cat: